core/bootstrap/src/main/java/org/jboss/modules/maven/MavenArtifactUtil.java

```diff
@@ -18,6 +18,15 @@
 
```

Comment:
I reverted the changed imports. I agree, it would be better to have a super class with the shared functionality. For that we have to rename the MavenResolver interface to ArtifactResovler and move it to an other package. 
From my perspective, this will be a big refactoring and I would prefer to do it later in a separate pull request.